src/main/kotlin/org/rust/lang/core/types/ty/Ty.kt

```diff
@@ -85,3 +88,30 @@ fun mergeFlags(element: BoundElement<*>): TypeFlags =
 
 fun mergeFlags(tys: List<Ty>): TypeFlags =
     tys.fold(0) { a, b -> a or b.flags }
+
+tailrec fun Ty.isSized(): Boolean {
```

Comment:
I don't really like that this function is public but I haven't come up with how to make it private because I use it in two places

Reply:
That's fine I think.